contact.md

```diff
@@ -9,22 +9,48 @@ menu: main
 
 *  Chat with us at [Gitter][webchat]
 *  Found a bug in UltraStar Deluxe? Use the [bugtracker][bugtracker] to create a ticket!
-*  Please do not try email support, there will be no response, please use the forums or bugtracker or the chat instead. Thank you...
+*  Please do not try email support, there will be no response, please use the forums or bugtracker or the chat instead. Thank you.
 
-## Forums
+## Communities & Forums
 
+* [Official Discord Community][official_discord]
 * [spanish users forum][forum_es]
-* [french users forum][forum_fr]
 * [english UltraStar asia forum][forum_asia]
 
+
 ## Social
 
 You can follow us on social network:
 
 *  {% include icon-twitter.html username="UltraStarPlay" %}
 
+
+## Help & Support
+
+* Help yourself with the [FAQ on open-music-games.org][open_music_games_faq]. It's a big FAQ about UltraStar, Vocaluxe & Co. 50 Questions about everything: Communities, Songs, Issues, Hardware, Setup, Contribution and future developement. Even a Song Creators FAQ and a Developer FAQ.
+* You can also get help from players in the [offical Discord Community "UltraStar, Vocaluxe & Co."][official_discord]. Just go to the channel "help-support".
+
+
+## Our partners
+
+We're happy to team up with:
+
+* [open-music-games.org][open_music_games_org] - central hub of open source music games founded in December 2022
+* [ultrastar-play.com][ultrastar_play_com] - renewal of UltraStar Deluxe with Unity Engine
+* [vocaluxe.org][vocaluxe_org] - the original successor of UltraStar Deluxe written in C# with MS Visual Studio
+* [performous.org][performous_org] - the all-in-one music game for singing, instruments and dancing in C++
+* [melodymania.org][melodymania_org] - commercial fork of UltraStar Play on Steam to finance developement of UltraStar Play
```

Comment:
I'm not really a fan of these four links. Isn't the first one pretty much dead and what's the point/goal of having the other three on an UltraStar-centered website? As far as I've been able to find, they're not linking back to USDX either, so why should we?

The other two (open music games and the format) are fine because they're not catering to one or the other.

Reply:
@barbeque-squared : I can understand your doubt, however I would like to see at least one or three links for following reasons:

1.  Ultrastar Play did get a big [merge back from melody mania](https://github.com/UltraStar-Deluxe/Play/pull/433) in the start of july 2024. everyone was waiting for that merge back. And now it happend. It's highly likeable now to find some unity developers to complete Ultrastar Play.

2. Performous Homepage already linked back to usdx.eu for years but to the old source forge repository of usdx. Look under "other games" [performous.org/others](http://performous.org/others ). They update the link, when I do a PR. I have a good connection to the leads.

3. Vocaluxe.org will also add a link to usdx and partners when I do a PR for the website. I have good connection to the leads too and they're not enemies of USDX ;)

4. The users get full transparency with the "partner links". Plus they see that all teams co-ooperate meaningful together for them to end any confusion.

5. In the end all the open source singing games are no enemies (as open-music-games.org shows) - they complement each other, focusing on their specific features. We as a whole big team should give players a curated portfolio of different singing games (with specific gameplay, partymodes, design style, game engines behind, etc.) instead of having only one popular game that is overloaden with features. Many games already share a common song format standard and a common txt-file hosting.

BTW: There is already a link to UltraStar Play Twitter Account. If we don't link to USP homepage, then we should remove the twitter link too.

Reply:
You make good points! I'm not really against adding them, but I want to clean up the pages a bit too. For example, the USPlay Twitter feels unnecessary if we just link to the USPlay website (or we just put it near that website link instead of its own block). Another example, I haven't looked at the gitter in years, I'd not be against keeping it alive for those that don't like Discord, but at this point we might need a gitter-discord bridge because most of us are there.

One of the links was also missing (I think vocaluxe or performous). The quick merge without them was because of time constraints: there's a big event happening this weekend which uses USDX, so I thought it would be useful to at least have the Discord links somewhere.

I'll make a new ticket to redo the pages a bit in general that also links to your comment so we don't forget adding back the links.